### PR TITLE
Update on getCode RPC function

### PR DESCRIPTION
**Pull request by vibenedict**
Closes https://github.com/keep-starknet-strange/rosettanet/issues/67

### CODE REVIEW

File: src/rpc/calls/getCode.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import { callStarknet } from '../../utils/callHelper'
+import { validateEthAddress } from '../../utils/validations'
+import {
+  getSnAddressFromEthAddress,
+} from '../../utils/wrapper'
+
+export async function getCodeHandler(
+  request: RPCRequest,
+): Promise<RPCResponse | RPCError> {
+  if (request.params.length == 0) {
```
Comment (AryanGodara) [inline src/rpc/calls/getCode.ts:11]:
```
if (request.params.length != 2)
```
If the function requires a specific number of parameters, just check for that ;)

File: tests/rpc/calls/getCode.test.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    const request = {
+      jsonrpc: '2.0',
+      method: 'eth_getCode',
+      params: ['0x94750381bE1AbA0504C666ee1DB118F68f0780D4', 960987],
+      id: 0,
+    }
+
+    const starknetRes: RPCResponse = <RPCResponse>await getCodeHandler(request)
+
+    expect(typeof starknetRes.result).toBeNull
```
Comment (AryanGodara) [inline tests/rpc/calls/getCode.test.ts:31]: This approach isn't correct. You're putting the `toBeNull` method over the `expect()` block.
So, even if you remove the null check, i.e.,
```
expect(typeof starknetRes.result)
expect(starknetRes.result)
```
Tests still pass :)

Instead, check if the result is null, or better yet, you can match the entire result object like this
```
expect(starknetRes).toMatchObject({
  jsonrpc: '2.0',
  id: 0,
  result: undefined,
});
```

File: tests/rpc/calls/getCode.test.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    const request = {
+      jsonrpc: '2.0',
+      method: 'eth_getCode',
+      params: ['0x94750381bE1AbA0504C666ee1DB118F68f0780D4', -1],
+      id: 0,
+    }
+
+    const starknetRes: RPCResponse = <RPCResponse>await getCodeHandler(request)
+
+    expect(typeof starknetRes.result).toBeNull
```
Comment (AryanGodara) [inline tests/rpc/calls/getCode.test.ts:45]: Same fix as above

**Review by AryanGodara — COMMENTED** [review]
Small fixes in tests

**Comment by AryanGodara** [comment]
> Update for this issue: #67



> Update for this issue: #67

Write `closes #67` so github can detect you're working on that issue ;)

**Comment by ermvrs** [comment]
please fix ci error

**Comment by vibenedict** [comment]
Fixed @ermvrs

**Comment by ermvrs** [comment]
fix ci error

**Comment by vibenedict** [comment]
Done @ermvrs

**Pull request merged**